crates/re_log_types/src/time.rs

```diff
@@ -156,6 +155,18 @@ impl TryFrom<std::time::SystemTime> for Time {
     }
 }
 
+// On non-wasm32 builds, `instant::SystemTime` is a re-export of `std::time::SystemTime`,
+// so it's covered by the above `TryFrom`.
+#[cfg(target_arch = "wasm32")]
```

Comment:
So remove `impl TryFrom<std::time::SystemTime> for Time {` and have just `impl TryFrom<instant::SystemTime> for Time {` ?

Reply:
Yeah, still likely kind of a stretch -- would have to be an external user using SystemTime and compiling to wasm and wanting to interact with our rust APIs...